ambry-commons/src/test/java/com.github.ambry.commons/BlobIdTest.java

```diff
@@ -562,6 +577,14 @@ private void assertBlobIdFieldValues(short version, BlobId blobId, BlobIdType ty
         assertEquals("Wrong container id in blobId: " + blobId, containerId, blobId.getContainerId());
         assertEquals("Wrong isEncrypted value in blobId: " + blobId, isEncrypted, BlobId.isEncrypted(blobId.getID()));
         break;
+      case BLOB_ID_V5:
+        assertEquals("Wrong type in blobId: " + blobId, type, blobId.getType());
+        assertEquals("Wrong datacenter id in blobId: " + blobId, datacenterId, blobId.getDatacenterId());
+        assertEquals("Wrong account id in blobId: " + blobId, accountId, blobId.getAccountId());
+        assertEquals("Wrong container id in blobId: " + blobId, containerId, blobId.getContainerId());
+        assertEquals("Wrong isEncrypted value in blobId: " + blobId, isEncrypted, BlobId.isEncrypted(blobId.getID()));
+        assertEquals("Wrong blobDataType value in blobId: " + blobId, blobDataType, blobId.getBlobDataType());
```

Comment:
should probably check that this is null in the other versions?

Reply:
ok